src/app/market/buy/checkout-process/checkout-process.component.ts

```diff
@@ -87,7 +87,7 @@ export class CheckoutProcessComponent implements OnInit {
 
   removeFromCart(shoppingCartId: number): void {
     this.cartService.removeItem(shoppingCartId).take(1)
-      .subscribe(this.getCart);
+      .subscribe(this.getCart.bind(this));
```

Comment:
.bind(this) shouldn't be required afaik?

Reply:
it is required I guess as it doesn't bind the component if you called without that iirc